app/src/main/java/com/willowtree/vocable/utils/FaceTrackingManager.kt

```diff
@@ -121,7 +121,6 @@ class FaceTrackingManager(
         if (ArCoreApk.getInstance().checkAvailability(activity) === ArCoreApk.Availability.UNSUPPORTED_DEVICE_NOT_CAPABLE) {
             Timber.e("TAG", "Augmented Faces requires ARCore.")
             Toast.makeText(activity, "Augmented Faces requires ARCore", Toast.LENGTH_LONG).show()
-            activity.finish()
```

Comment:
The method name shouldn't have finish in it anymore if the method no longer finishes the Activity